File: app/src/main/java/br/com/yourapp/pokeapi/models/GameIndex.java
package br.com.yourapp.pokeapi.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class GameIndex implements Serializable {
    @SerializedName("game_index")
    public String game_index;

    @SerializedName("url")
    public List<Version> url;
}

File: app/src/main/java/br/com/yourapp/pokeapi/models/GameIndex.kt
package br.com.yourapp.pokeapi.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class GameIndex : Serializable {
    @SerializedName("game_index")
    var game_index: String? = null

    @SerializedName("url")
    var url: List<Version>? = null
}